packages/web/integrations/notifi/notifi-popover.tsx

```diff
@@ -96,7 +96,10 @@ export const NotifiPopover: FunctionComponent<NotifiButtonProps> = ({
           <NotifiIconButton
             className={className}
             hasUnreadNotification={hasUnreadNotification}
-            onClick={() => logEvent([EventName.Notifications.iconClicked])}
+            onClick={() => {
+              if (isOverLayEnabled) setIsOverLayEnabled(false);
+              logEvent([EventName.Notifications.iconClicked]);
+            }}
```

Comment:
Fix the issue that keeps the overlay open when closing notifications by clicking on the bell button.